examples/howto/data_container.myst.md

```diff
@@ -118,25 +115,25 @@ We can get the data container variable from the model using:
 model["data"].get_value()
 ```
 
-Note that we used a theano method {meth}`theano.compile.sharedvalue.SharedVariable.get_value` of class {class}`theano.compile.sharedvalue.SharedVariable` to get the value of the variable. This is because our variable is actually a `SharedVariable`.
+Note that we used a pytensor method {meth}`pytensor.compile.sharedvalue.SharedVariable.get_value` of class {class}`pytensor.compile.sharedvalue.SharedVariable` to get the value of the variable. This is because our variable is actually a `SharedVariable`.
 
 ```{code-cell} ipython3
 type(data)
 ```
 
 The methods and functions related to the Data container class are:
 
-- `data_container.get_value` (method inherited from the theano SharedVariable): gets the value associated with the `data_container`.
-- `data_container.set_value` (method inherited from the theano SharedVariable): sets the value associated with the `data_container`.
-- {func}`pymc.set_data`: PyMC3 function that sets the value associated with each Data container variable indicated in the dictionary `new_data` with it corresponding new value.
+- `data_container.get_value` (method inherited from the pytensor SharedVariable): gets the value associated with the `data_container`.
+- `data_container.set_value` (method inherited from the pytensor SharedVariable): sets the value associated with the `data_container`.
+- {func}`pymc.set_data`: PyMC function that sets the value associated with each Data container variable indicated in the dictionary `new_data` with it corresponding new value.
 
 +++
 
 ## Using Data container variables to fit the same model to several datasets
 
-This and the next sections are an adaptation of the notebook ["Advanced usage of Theano in PyMC3"](../Advanced_usage_of_Theano_in_PyMC3.html#using-shared-variables) using `pm.Data`.
+This and the next sections are an adaptation of the notebook ["Advanced usage of pytensor in PyMC"](../Advanced_usage_of_pytensor_in_PyMC.html#using-shared-variables) using `pm.MutableData`.
```

Comment:
This page no longer exists as it didn't makes sense to keep around. I think we can delete this sentence altogether.

Reply:
Removed